src/handy.jl

```diff
@@ -279,8 +279,10 @@ Get a `MYSQL_BIND` instance given the mysql type `typ` and a `value`.
 mysql_bind_init(typ::MYSQL_TYPE, value) =
     mysql_bind_init(mysql_get_julia_type(typ), typ, value)
 
-mysql_bind_init(jtype::Union{Type{Date}, Type{DateTime}}, typ, value) =
-    MYSQL_BIND([convert(MYSQL_TIME, convert(jtype, value))], typ)
+mysql_bind_init(jtype::Type{Union{Date, Missings.Missing}}, typ, value) =
```

Comment:
Since `Missing` is in Base in Julia 0.7, it would make sense to do `using Missings: missing` at the top of the file/module so that you just have to remove one line once you drop 0.6 support.